sources/Util/Utils.cpp

```diff
@@ -7,8 +7,12 @@
 
 #include "Vector/Vector3.hpp"
 #include "Models/Vector3.hpp"
+#include "Utils.hpp"
 
-RAY::Vector3 toRAY(const WAL::Vector3f &wal)
+namespace BBM
 {
-	return RAY::Vector3(wal.x, wal.y, wal.y);
+	RAY::Vector3 Utils::toRAY(const BBM::Vector3f &wal)
```

Comment:
This function could be a conversion operator inside the Vector3 class (at first it was a function because the Vector3 was inside the WAL library but since the Vector has been moved, this function has no reason to exist outside of the class).

Reply:
As you please -> https://github.com/AnonymusRaccoon/Bomberman/pull/64/commits/e70dbd5b3b428d516d2e190c2e117f6eed62b01b